lib/csv_to_popolo.rb

```diff
@@ -339,30 +353,72 @@ def given?(key)
       @r.key?(key) && !@r[key].nil? && !@r[key].empty?
     end
 
+    def split_cell(key)
```

Comment:
`split_cell` sounds a little bit more like an action to me (and a potentially destructive one at that, though the absence of a ! on the end of the name helps mitigate that a bit). But really all this is doing is telling you what's in the cell — perhaps this would be better named as something like `cell_values`?